google-cloud-pubsub/lib/google/cloud/pubsub/v1/subscriber_api.rb

```diff
@@ -534,6 +536,7 @@ def acknowledge \
               ack_ids: ack_ids
             }.delete_if { |_, v| v.nil? })
             @acknowledge.call(req, options)
+            nil
```

Comment:
That doesn't matter I think. If you want to push your changes to this PR, please do that.